### PR TITLE
have quitteSort() sort factor levels alphabetically

### LINKED ISSUE #73: in quitteSort, sort levels alphabetically

### PR DESCRIPTION
**Pull request by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q**
close #73

### CODE REVIEW

File: .buildlibrary [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ValidationKey: '613522360'
+ValidationKey: '6.14e+08'
```
Comment (orichters) [inline .buildlibrary:1]: never seen scientific notation here :D

**Review by orichters — APPROVED** [review]

**Comment by orichters** [comment]
Thanks for fixing it!

File: .buildlibrary (hunk 1 above)
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [reply .buildlibrary:1]: :rofl:

Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [reply .buildlibrary:1]: Turns out, today is a good day for that to happen, and version numbers ending on the digit 0 or five help too.

**Pull request merged**